pkgs/development/libraries/jemalloc/default.nix

```
@@ -19,6 +20,14 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-LbgtHnEZ3z5xt2QCGbbf6EeJvAU3mDw7esT3GJrs/qo=";
   };
 
+  patches = [
+    # fix tests under --with-jemalloc-prefix=, see https://github.com/jemalloc/jemalloc/pull/2340
+    (fetchpatch {
+      url = "https://github.com/jemalloc/jemalloc/commit/d00ecee6a8dfa90afcb1bbc0858985c17bef6559.patch";
+      sha256 = "sha256:11wp6byrhxi8ymqgknb40xx5xcwyj60ydajqh2029qc924ivi61p";
```

If you're going to use `sha256` and base-32 hashes, we can drop the `sha256:` prefix. (Though perhaps it should be converted to SRI, which would maintain consistency with the source hash?)

```suggestion
      sha256 = "11wp6byrhxi8ymqgknb40xx5xcwyj60ydajqh2029qc924ivi61p";
```

---

SRI looks simpler, though a little confused on raw hash vs sri hash, looks like both "sha256:" and "sha256-" are accepted by `fetchurl` and `fetchpatch`

https://nixos.wiki/wiki/Nix_Hash says "sha256:" is dangerous or something with no clear action.
https://github.com/NixOS/nixpkgs/blob/master/pkgs/build-support/fetchurl/default.nix#L61 also used by `fetchpatch` says `hash` is an `SRI` hash
https://github.com/NixOS/nix/blob/646af7325d93f98802b989f8a8e008a25f7a4788/src/libutil/hash.cc#L154 but looks like `outputHash` can handle raw "sha256:" hashes too